HHS/Head-Start-TTADP
# [PROD] 03/22 TTAHUB-2742 Fix MyReports filter cache, TTAHUB-2745 Prevent saving of session reports in event blob

## Description of change

**TTAHUB-2742:** Fixes a bug where my reports filter result was being cached for all users.
**TTAHUB-2745:** Fixes a bug where the session report data was being saved in the blob for the event causing large payload.

## How to test

**TTAHUB-2742:** Make sure the MyReports filter is showing different data for different users.
**TTAHUB-2745:** Make sure you can save event 1305.

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-2742
* https://jira.acf.gov/browse/TTAHUB-2745


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
